### PR TITLE
Upgrade to graphql-codegen version 1.7.1

### PR DESCRIPTION
**Pull request by cdupuis**

### CODE REVIEW

File: bin/gql-gen.ts [hunk 1]
```diff
@@ -34,26 +43,6 @@ async function libDir(cwd: string): Promise<string> {
     }
 }
 
-/**
- * Patch the handlebar template that renders the generated types out.
- * This patch puts back the original behaviour of creating only optional properties on generated types.
```
Comment (ddgenome) [inline bin/gql-gen.ts]: Nice if this is no longer necessary.

File: bin/gql-gen.ts [hunk 2]
```diff
@@ -67,38 +56,65 @@ async function main(): Promise<void> {
...
+                const document = parse(content);
+                documents.push({
+                    content: document,
+                    filePath: graphqlFile,
+                });
+            }
+            config.documents = documents;
+
+            // Make all properties optional to retain backwards compatibility
+            const typesContent = (await codegen(config)).replace(/ ([a-zA-Z_\-0-9]+): Maybe/g, ` $1?: Maybe`);
```
Comment (ddgenome) [inline bin/gql-gen.ts]: Or not

Comment (ddgenome) [reply bin/gql-gen.ts]: The `Maybe` doesn't accomplish the same?

**Comment by cdupuis** [comment]
Pull request auto merged by Atomist.

* 1 approved review by @ddgenome
* 2 successful checks

[atomist:generated] [auto-merge:on-approve]

**Pull request merged**